docs/customize/administration_interface.md

```diff
@@ -0,0 +1,398 @@
+# Administration panel views
+
+## Summary
+
+The following document is a developer guide to the invenio administration panel, detailing its programmatic interface and usage.
+
+## Intended audience
+
+This guide is intended for system administrators and developers of InvenioRDM.
```

Comment:
I think in other docs we dont put intended audience (e.g. not like an RFC). I think is implicit in "customize" for sysadmins and "develop" for developers. But in any case, if stays:

```suggestion
This guide is intended for system administrators of InvenioRDM.
```